server/tnt_workdir/api/cpapi.lua

```diff
@@ -134,6 +134,7 @@ function getNearbyCashpoints(reqJson)
         matchingRubFilter,
         matchingUsdFilter,
         matchingEurFilter,
+        matchingTimeFilter,
```

Comment:
Why added exactly here? Checks are performed in order of their appearance in `filtersList`. I think there is an ability to increase perf by applying time filter after all other filters. Because most of all `match*` functions are lightweight and check only matching of particular field to one in `req.filter`. On the other hand `matchingTimeFilter` is quite complex and there is reason to do it on as less as possible count of cashpoints. If there is any reason to add it here please explain. Thanks